src/components/datePicker/DatePickerReducers.ts

```diff
@@ -0,0 +1,111 @@
+import { IReduxAction } from '../../utils/ReduxUtils';
+import { IReduxActionsPayload } from '../../ReactVapor';
+import { DatePickerActions } from './DatePickerActions';
+import * as _ from 'underscore';
+
+export interface IDatePickerState {
+  id: string;
+  calendarId: string;
+  color: string;
+  lowerLimit: Date;
+  upperLimit: Date;
+  isRange: boolean;
+  selected: string;
+  appliedLowerLimit: Date;
+  appliedUpperLimit: Date;
+}
+
+export const datePickerInitialState: IDatePickerState = {
+  id: undefined,
+  calendarId: undefined,
+  color: undefined,
+  isRange: false,
+  lowerLimit: new Date(new Date().setHours(0, 0, 0, 0)),
+  upperLimit: new Date(new Date().setHours(23, 59, 59, 999)),
```

Comment:
I think the moment functions are easier to understand: `moment().startOf('day').toDate()` & `moment().endOf('day').toDate()`